src/edu/csus/ecs/pc2/ui/JSerializFilePicker.java

```diff
@@ -76,8 +76,8 @@ private void initialize() {
     private JButton getPickFileButton() {
         if (pickFileButton == null) {
             pickFileButton = new JButton();
-            pickFileButton.setName("Select");
-            pickFileButton.setText("Select");
+            pickFileButton.setName("Select...");
```

Comment:
I don't see any reason to change the "Name" of the button.  Select was probably fine for the name.  However, that being said, I don't think it matters here since no one looks up the component by name.  I'm fine leaving as "Select...", or, removing that line altogether.

Reply:
Removed the line then build and ran the JUnits again.